### ISSUE TITLE
NullPointerException - InventoryListener

### ISSUE THREAD

**Issue by CaptainObvious0** [CaptainObvious0/NoCheatPlus#15]
NullPointerException Errors will appear in console due to added 'chestOpen' event. This will be fixed later as this serves here as a reminder for me.

**Comment by mibby** [comment]
```
[01:47:10] [Craft Scheduler Thread - 82/ERROR]: [NoCheatPlus] Exception with AutoListener(fr.neatmonster.nocheatplus.checks.inventory.InventoryListener.chestOpen/org.bukkit.event.player.PlayerInteractEvent/MONITOR), processing org.bukkit.event.player.PlayerInteractEvent: InvocationTargetException -> log once id=494178945
[01:47:10] [Craft Scheduler Thread - 82/ERROR]: [NoCheatPlus] Exception:
java.lang.reflect.InvocationTargetException
	java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	java.base/java.lang.reflect.Method.invoke(Method.java:566)
	fr.neatmonster.nocheatplus.event.mini.MultiListenerRegistry$AutoListener.onEvent(MultiListenerRegistry.java:82)
	fr.neatmonster.nocheatplus.event.mini.MiniListenerNode.onEvent(MiniListenerNode.java:157)
	fr.neatmonster.nocheatplus.event.mini.EventRegistryBukkit$4.execute(EventRegistryBukkit.java:124)
	co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:80)
	org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:66)
	org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:520)
	org.bukkit.craftbukkit.v1_13_R2.event.CraftEventFactory.callPlayerInteractEvent(CraftEventFactory.java:320)
	org.bukkit.craftbukkit.v1_13_R2.event.CraftEventFactory.callPlayerInteractEvent(CraftEventFactory.java:287)
	org.bukkit.craftbukkit.v1_13_R2.event.CraftEventFactory.callPlayerInteractEvent(CraftEventFactory.java:283)
	net.minecraft.server.v1_13_R2.PlayerConnection.a(PlayerConnection.java:1385)
	net.minecraft.server.v1_13_R2.PacketPlayInBlockPlace.a(PacketPlayInBlockPlace.java:26)
	net.minecraft.server.v1_13_R2.PacketPlayInBlockPlace.a(PacketPlayInBlockPlace.java:5)
	net.minecraft.server.v1_13_R2.PlayerConnectionUtils.lambda$ensureMainThread$0(PlayerConnectionUtils.java:15)
	java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	net.minecraft.server.v1_13_R2.SystemUtils.a(SystemUtils.java:108)
	net.minecraft.server.v1_13_R2.MinecraftServer.b(MinecraftServer.java:1017)
	net.minecraft.server.v1_13_R2.DedicatedServer.b(DedicatedServer.java:439)
	net.minecraft.server.v1_13_R2.MinecraftServer.a(MinecraftServer.java:939)
	net.minecraft.server.v1_13_R2.MinecraftServer.run(MinecraftServer.java:836)
	java.base/java.lang.Thread.run(Thread.java:834)
caused by:
java.lang.NullPointerException
	fr.neatmonster.nocheatplus.checks.inventory.InventoryListener.chestOpen(InventoryListener.java:319)
	java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	java.base/java.lang.reflect.Method.invoke(Method.java:566)
	fr.neatmonster.nocheatplus.event.mini.MultiListenerRegistry$AutoListener.onEvent(MultiListenerRegistry.java:82)
	fr.neatmonster.nocheatplus.event.mini.MiniListenerNode.onEvent(MiniListenerNode.java:157)
	fr.neatmonster.nocheatplus.event.mini.EventRegistryBukkit$4.execute(EventRegistryBukkit.java:124)
	co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:80)
	org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:66)
	org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:520)
	org.bukkit.craftbukkit.v1_13_R2.event.CraftEventFactory.callPlayerInteractEvent(CraftEventFactory.java:320)
	org.bukkit.craftbukkit.v1_13_R2.event.CraftEventFactory.callPlayerInteractEvent(CraftEventFactory.java:287)
	org.bukkit.craftbukkit.v1_13_R2.event.CraftEventFactory.callPlayerInteractEvent(CraftEventFactory.java:283)
	net.minecraft.server.v1_13_R2.PlayerConnection.a(PlayerConnection.java:1385)
	net.minecraft.server.v1_13_R2.PacketPlayInBlockPlace.a(PacketPlayInBlockPlace.java:26)
	net.minecraft.server.v1_13_R2.PacketPlayInBlockPlace.a(PacketPlayInBlockPlace.java:5)
	net.minecraft.server.v1_13_R2.PlayerConnectionUtils.lambda$ensureMainThread$0(PlayerConnectionUtils.java:15)
	java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	net.minecraft.server.v1_13_R2.SystemUtils.a(SystemUtils.java:108)
	net.minecraft.server.v1_13_R2.MinecraftServer.b(MinecraftServer.java:1017)
	net.minecraft.server.v1_13_R2.DedicatedServer.b(DedicatedServer.java:439)
	net.minecraft.server.v1_13_R2.MinecraftServer.a(MinecraftServer.java:939)
	net.minecraft.server.v1_13_R2.MinecraftServer.run(MinecraftServer.java:836)
	java.base/java.lang.Thread.run(Thread.java:834)

```

**Comment by CaptainObvious0** [comment]
Fixed with ff6b47a
